content/en/data_security/_index.md

```diff
@@ -40,6 +40,8 @@ Datadog allows you to send data to Datadog in multiple ways, including from the
 
 Data in motion through Datadog provided tools is protected with TLS and HSTS. Data stored by Datadog is protected by encryption, access controls, and authentication. For specifics, read more at [Datadog Security][1].
 
+Starting **July 1st, 2024**, Datadog will force HTTP to HTTPS redirection for all intake endpoints, using an HTTP 308 response code. This change is necessary as some intake endpoints still accept plain HTTP payloads to maintain compatibility with older API clients. Ensure that your application clients can follow these redirection codes, or update their configuration to use HTTPS. Otherwise, you might receive connection error messages.
+
```

Comment:
can you add the rest of the message written here https://docs.google.com/document/d/1IZucDp2NGBt5HIYSj84fSozInhn5ENCKzv3RqjjwMXU/edit?pli=1#heading=h.5kijwij0wyic:

> Our internal telemetry indicates that the majority of HTTP requests come from the Fluent Bit application. Fluent Bit can be configured to use HTTPS using the TLS configuration parameter. See [the Fluent Bit integration](https://docs.datadoghq.com/integrations/fluentbit/). 
For any additional questions, please reach out to our [support team](https://docs.datadoghq.com/help/).